prometheus-to-sd/main.go

```diff
@@ -66,7 +66,12 @@ var (
 	projectOverride = flag.String("project-id", "",
 		"GCP project to send metrics to. If not set, defaults to value from GCE Metadata Server.")
 	monitoredResourceTypePrefix = flag.String("monitored-resource-type-prefix", "", "MonitoredResource type prefix, to be appended by 'container', 'pod', and 'node'.")
-	monitoredResourceLabels     = flag.String("monitored-resource-labels", "", "Manually specified MonitoredResource labels. It is in URL parameter format, like 'A=B&C=D&E=F'. When this field is specified, 'monitored-resource-type-prefix' is also required. To note: 'namespace-name', 'pod-name', and 'container-name' should not be provided in this flag and they will always be overwritten by values in other command line flags.")
+	monitoredResourceLabels     = flag.String("monitored-resource-labels", "", "Manually specified MonitoredResource labels. " +
+	"It is in URL parameter format, like 'A=B&C=D&E=F'. " +
```

Comment:
These strings should be indented more than the enclosing statement.

Reply:
Done.